contrib/packages/debian/rules

```diff
@@ -4,7 +4,7 @@
 DEB_PYTHON_SYSTEM := pysupport
 
 include /usr/share/cdbs/1/rules/debhelper.mk
-include /usr/share/cdbs/1/class/python-distutils.mk
+#include /usr/share/cdbs/1/class/python-distutils.mk
```

Comment:
Why you comment on this and not deleted it?

Reply:
I wasn't as sure about the rules file as much as I was for the Python files. If they require removal, I'd do right away.

Reply:
I'm not so familiar with CDBS in debian rules and how this will affect this migration. Yes, afaict, we need to remove this, but just need to make sure that debian packages are still working or if we need to add some rules for setuptools too.

Reply:
`setuptools` are the defacto Python packaging standard as of 3.8 so I do not think we need to explicitly specify about it in the Debian rules (unless I figured the way that file works wrongly)